core/conf/application-connection-types.yml

```diff
@@ -64,14 +64,18 @@ iesi:
         parameters:
```

Comment:
is the content of this file used in all the connection-type yaml's? (core, core test, rest, rest test)?

Reply:
Yes